radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/discovery/PeerDiscovery.java

```diff
@@ -97,10 +98,11 @@
 public final class PeerDiscovery {
   private static final Logger log = LogManager.getLogger();
 
-  private static final int MAX_PEERS_IN_RESPONSE = 50;
+  private static final long MAX_PEERS_IN_RESPONSE = 50;
```

Comment:
I would keep it as `int` and convert `MAX_PEERS_IN_RESPONSE - 1` to `long`.

Reply:
that, on the other hand, makes intellij complain about unneeded type cast